package.json

```diff
@@ -55,7 +57,6 @@
     "pretty-format": "26.6.2",
     "raw-loader": "^0.5.1",
     "react-dnd-test-backend": "^9.4.0",
-    "react-dnd-test-utils": "^9.4.0",
```

Comment:
If this is an unused dependency, can we already drop it today in a separate PR?

Reply:
I'll check :+1:

Reply:
https://github.com/theforeman/foreman/pull/10020